azurerm/resource_arm_devspace_controller.go

```diff
@@ -171,21 +171,19 @@ func resourceArmDevSpaceControllerRead(d *schema.ResourceData, meta interface{})
 		d.Set("location", azureRMNormalizeLocation(*location))
 	}
 
-	d.Set("sku", flattenDevSpaceControllerSku(result.Sku))
+	if err := d.Set("sku", flattenDevSpaceControllerSku(result.Sku)); err != nil {
+		return fmt.Errorf("Error flattenning Sku for DevSpace Controller %q (Resource Group %q): %+v", name, resGroupName, err)
```

Comment:
this would be failing to set the schema field, so we should make this:

```suggestion
		return fmt.Errorf("Error flattening`sku`: %+v", err)
```